### ISSUE TITLE
Install Primer3 on Windows 

### ISSUE THREAD

**Issue by stefandiederich** [primer3-org/primer3#18]
Hi, 

I have some trouble installing Primer3 version 2.4.0 on an Windows 10 system. I downloaded the ZIP file and extracted it. After that I changed folder to "test" and I run the command `mingw32-make TESTOPTS=--windows`
Then I got the following error message:

```C:\Users\die9s\Desktop\primer3-2.4.0\test>mingw32-make TESTOPTS=--windows
cd ..\src & mingw32-make
mingw32-make[1]: Entering directory 'C:/Users/die9s/Desktop/primer3-2.4.0/src'
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  primer3_boulder_main.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  -o format_output.o format_output.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  -o read_boulder.o read_boulder.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  -o print_boulder.o print_boulder.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  masker.c
masker.c:8:22: fatal error: sys/mman.h: No such file or directory
compilation terminated.
Makefile:226: recipe for target 'masker.o' failed
mingw32-make[1]: *** [masker.o] Error 1
mingw32-make[1]: Leaving directory 'C:/Users/die9s/Desktop/primer3-2.4.0/src'
Makefile:94: recipe for target 'makeexes' failed
mingw32-make: *** [makeexes] Error 2
```

Do you have any idea solving this problem?
Thanks in advance
Stefan

**Comment by untergasser** [comment]
Hi Stefan,
I fixed the latest code, so you could try directly from GitHub. Let me know if the problem still exists, be aware that the masker is not available on windows versions.
Best,
Andreas

**Comment by liucyuic** [comment]
I also have problem installing it on Win 10:
 make
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  primer3_boulder_main.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  -o format_output.o format_output.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  -o read_boulder.o read_boulder.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  -o print_boulder.o print_boulder.c
g++ -c -g -Wall -D__USE_FIXED_PROTOTYPES__ -O2  masker.c
In file included from /usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/i686-pc-cygwin/bits/gthr.h:148:0,
                 from /usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/ext/atomicity.h:35,
                 from /usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/bits/ios_base.h:39,
                 from /usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/ios:42,
                 from /usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/istream:38,
                 from /usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/fstream:38,
                 from masker.c:1:
/usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/i686-pc-cygwin/bits/gthr-default.h: In function ‘int __gthread_mutex_timedlock(__pthread_mutex_t**, const __gthread_time_t*)’:
/usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/i686-pc-cygwin/bits/gthr-default.h:768:12: error: ‘pthread_mutex_timedlock’ was not declared in this scope
     return __gthrw_(pthread_mutex_timedlock) (__mutex, __abs_timeout);
            ^
/usr/lib/gcc/i686-pc-cygwin/7.3.0/include/c++/i686-pc-cygwin/bits/gthr-default.h:768:12: note: suggested alternative: ‘__gthread_mutex_timedlock’
make: *** [Makefile:226: masker.o] Error 1

**Comment by untergasser** [comment]
Please try Release version 2.5.0 or download binary from release.

Best,

Andreas
